prisma/schema.prisma

```diff
@@ -101,3 +104,50 @@ enum Status {
   RSVP
   CHECKED_IN
 }
+
+model DH10Application {
+  id String @id @default(cuid())
+
+  firstName String
+  lastName  String
+
+  birthday DateTime
```

Comment:
Should this be DateTime? @db.Date would be better I think...

https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#cockroachdb-6